mods/tsumeta/items.js

```diff
@@ -808,6 +809,24 @@ exports.BattleItems = {
 			}
 			return true;
 		},
+		onStart: function (pokemon) {
+			if (pokemon.baseTemplate.baseSpecies === 'Arceus' || pokemon.baseTemplate.baseSpecies === 'Dunsparce') {
+				let stats = [];
+				for (let stat in pokemon.boosts) {
+					if (stat !== 'accuracy' && stat !== 'evasion') {
+						stats.push(stat);
+					}
+				}
+				if (stats.length) {
+					let randomStat = stats[this.random(stats.length)];
+					let boost = {};
+					boost[randomStat] = 1.5;
+					return this.chainModify(boost);
```

Comment:
use `this.boost({stat: 1}, pokemon);`
Your current setup could remove boosts.

Reply:
Ah ok

Reply:
This is intended to be a modifier not a regular boost (more like Huge Power rather than say Moxie boosts)

Reply:
The description states it boosts a random stat, which would mean a boost like agility gives. Tagging @DesoGit for clarification.